src/flavors/DQMC/measurements/generic.jl

```diff
@@ -323,30 +326,30 @@ end
 # Call kernel for each site (linear index)
 @bm function apply!(iter::DirectLatticeIterator, measurement, mc::DQMC, model, packed_greens)
     for i in iter
-        measurement.temp[i] += measurement.kernel(mc, model, i, packed_greens)
+        measurement.temp[i] += Base.invokelatest(measurement.kernel, mc, model, i, packed_greens)
```

Comment:
I don't think I want invokelatest for something that runs this frequently. It's required to avoid world age errors when loading. If I can't get rid of it I'll probably scrap the idea of saving `Expr`

Reply:
Guess replacing a global function with `@eval` is fine?